src/components/product-social-links/product-social-links.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import { FacebookIcon, PinterestIcon, WhatsappIcon } from '../icons';
+import styles from './product-social-links.module.scss';
+
+interface ProductSocialLinksProps {
+    productCanonicalUrl: string;
+    className?: string;
+}
+
+export const ProductSocialLinks = ({ productCanonicalUrl, className }: ProductSocialLinksProps) => {
```

Comment:
Maybe `ShareProductLinks`? I was confused at the beginning why social links receive product URL, but it starts making sense if you know these are links for sharing the product rather than links to the product.